shardingsphere-integration-test/shardingsphere-proxy-docker-build/Dockerfile

```diff
@@ -20,6 +20,6 @@ FROM java:8
 
 ARG APP_NAME
 ADD target/${APP_NAME}.tar.gz /opt
-ADD src/main/resources/tool/wait-for-it.sh /opt
+ADD https://github.com/vishnubob/wait-for-it/blob/master/wait-for-it.sh /opt
```

Comment:
This link seems to be in html format, you may want to use https://raw.githubusercontent.com/vishnubob/wait-for-it/master/wait-for-it.sh ?